src/serac/physics/utilities/functional/tensor.hpp

```diff
@@ -1471,4 +1490,35 @@ auto chain_rule(const tensor<double, m, n, p...>& df_dx, const tensor<double, p.
   return total;
 }
 
+/**
+ * @brief tensor conversions since double and tensor are different types
+ */
```

Comment:
maybe call this `convert_to_tensor` or `convert_to_tensor_with_shape` (or even overload `make_tensor`), since `convert` doesn't really convey what is happening